### PR TITLE
Fixed a blocker bug in the scheduler: num_runs used where runs intended

### PR DESCRIPTION
**Pull request by r39132**
Dear Airflow Maintainers,

Please accept the following PR that
- Addresses the following issues (links to issues addressed by this PR) : https://github.com/airbnb/airflow/issues/1395

Reminder to contributors:
- You must add an Apache License header to all new files
- Please squash your commits when possible and follow the [7 rules of good Git commits](http://chris.beams.io/posts/git-commit/#seven-rules)

In a nutshell, `num_runs` used where `runs` intended, leading to a failure to load the dag bag.

### CODE REVIEW

**Comment by r39132** [comment]
FYI @bolkedebruin
